queue_services/entity-filer/src/entity_filer/filing_processors/continuation_out.py

```diff
@@ -0,0 +1,58 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the consent continuation out filing."""
+from contextlib import suppress
+from datetime import datetime
+from typing import Dict
+
+import dpath
+from legal_api.models import Business, Filing
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components import filings
+
+
+def process(business: Business, continuation_out_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the continuation out filing into the business model objects."""
+    # update continuation out, if any is present
+    with suppress(IndexError, KeyError, TypeError):
+        court_order_json = dpath.util.get(filing, '/continuationOut/courtOrder')
+        filings.update_filing_court_order(continuation_out_filing, court_order_json)
+
+    continuation_out_json = filing['continuationOut']
+
+    details = continuation_out_json.get('details')
+    legal_name = continuation_out_json.get('legalName')
+    continuation_out_date = datetime.strptime(continuation_out_json.get('continuationOutDate'), '%Y-%m-%d')
+    foreign_jurisdiction = continuation_out_json.get('foreignJurisdiction')
+    foreign_jurisdiction_country = foreign_jurisdiction.get('country')
+
+    continuation_out_filing.order_details = details
+
+    if continuation_out_date >= datetime.now():
+        business.state = Business.State.HISTORICAL
```

Comment:
you don't need this check(`if continuation_out_date >= datetime.now():`) as the filing would have passed legal api validation already.

Reply:
Make sense

Reply:
Remove L43(`if continuation_out_date >= datetime.now():`) and keep L44(`business.state = Business.State.HISTORICAL`)

Reply:
Sorry, I meant the legal-api validation part.

Reply:
can you reply to the appropriate comment?

Reply:
The filing json is validated against the schema  first here -> https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/filings/validations/validation.py#L51.  It only checks that the date is provided and that the date format is correct. 

After that, more detailed validation is done by the legal api -> https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/filings/validations/continuation_out.py#L40.